tools/remove_grouping_from_report_results.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require 'trollop'
+
+ARGV.shift if ARGV[0] == '--' # if invoked with rails runner
```

Comment:
Uh, who would really want to deal with a slow `-h` to do this...

Also, can this work like that?  I have had troubles in the past requiring `config/environment` twice in one script, but maybe this will be fine.

Reply:
This is a thing we've done since the dawn of time for tools.  I suppose it's fine to not do this and expect to be run with ruby.